azure-mgmt-network/HISTORY.rst

```diff
@@ -3,11 +3,36 @@
 Release History
 ===============
 
-1.8.0a1 (2017-XX-XX)
-++++++++++++++++++++
+2.0.0rc1 (2018-03-07)
++++++++++++++++++++++
+
+**Breaking changes**
+
+This version introduces a new generation of code generator that *might* introduce breaking changes. 
+
+- Model signatures are now using only keywords-arguments syntax. Every positional arguments are required to be rewritten as keywords-arguments.
```

Comment:
Model signatures now use only keyword-argument syntax. All positional arguments must be re-written as keyword-arguments.